near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -53,7 +51,7 @@ pub struct NonFungibleToken {
 
 #[derive(BorshStorageKey, BorshSerialize)]
 pub enum StorageKey {
-    TokensPerOwner { account_hash: Vec<u8> },
+    TokensPerOwner { account_id_hash: CryptoHash },
```

Comment:
Still am advocating for this to be an AccountId.  You charge for a full length account id, but it will reduce storage on average since most real users have shorter account ids, and it will not require a host call to produce a hash.

Reply:
Furthermore a state dump will make it clear who the owner is.

Reply:
This would be a state-breaking change, though. This can come if there are breaking changes or if the standards were re-written.